spec/unit/puppet_x/ioa_interface_spec.rb

```
@@ -161,6 +168,20 @@
         @transport.should_receive(:command).with("no vlan untagged")
         @interface.update(@old_params, @new_params)
       end
+
+      it 'should remove vlan as untagged if setting to tagged with inclusive flag true' do
+        @new_params[:vlan_tagged] = '18'
+        @new_params[:inclusive_vlans] = :true
+        @transport.should_receive(:command).with("no vlan untagged")
```

I guess we decide yesterday this should not remove the untagged vlan? Similar to above case I think it should raise an error then, and this test case needs to be updated.